jujugui/static/gui/src/app/components/machine-view/unplaced-unit/unplaced-unit.js

```diff
@@ -29,6 +29,17 @@ YUI.add('machine-view-unplaced-unit', function() {
     */
     beginDrag: function(props) {
       return {unit: props.unit};
+    },
+
+    /**
+      Called to check if the component is allowed to be dragged.
+      See: http://gaearon.github.io/react-dnd/docs-drag-source.html
+
+      @method canDrag
+      @param {Object} props The component props.
+    */
+    canDrag: function(props) {
```

Comment:
Yeah, it's one of the methods called by the ReactDnD module. They're a bit mysterious as it doesn't get called by any of our code, but rather used internally.